bootstrapper/misc/miscNg2Context.ts

```diff
@@ -2,14 +2,18 @@ import { Component, ViewChild } from '@angular/core';
 import { Observable, Subject } from 'rxjs';
 
 import { BusyComponent } from '../../source/components/busy/busy';
+import { RatingBarComponent } from '../../source/components/ratingBar/ratingBar';
+import { SpinnerComponent } from '../../source/components/inputs/index';
 
 @Component({
 	selector: 'tsMiscNgContext',
 	template: require('./miscNg2Context.html'),
-	directives: [BusyComponent],
+	directives: [BusyComponent, RatingBarComponent, SpinnerComponent],
 })
 export class MiscNgContextBootstrapper {
 	@ViewChild('busy2') busy2: BusyComponent;
+	width: number = 100;
+	value: number = 0;
 
 	wait(): Observable<void> {
 		const subject: Subject<void> = new Subject<void>();
```

Comment:
Good point. I think I do still like the redundancy of explicitly typing in the case of class properties, like the `width` and `value` above, but would be perfectly content to drop them for locals like this.